server/services/activitiesService.ts

```diff
@@ -0,0 +1,38 @@
+import { format } from 'date-fns'
+import ActivitiesApiClient from '../data/activitiesApiClient'
+import { HmppsUser } from '../interfaces/hmppsUser'
+import { ScheduledEvent } from '../@types/activitiesApi/types'
+import logger from '../../logger'
+
+export default class ActivitiesService {
+  constructor(private readonly activitiesApiClient: ActivitiesApiClient) {}
+
+  async getScheduledEventsByPrisonerNumbers(
+    prisonCode: string,
+    date: string,
+    prisonerNumbers: string[],
+    user: HmppsUser,
+  ): Promise<ScheduledEvent[]> {
+    const scheduledEvents = await this.activitiesApiClient.getScheduledEventsByPrisonerNumbers(
+      prisonCode,
+      format(date, 'yyyy-MM-dd'),
+      prisonerNumbers,
+      user,
+    )
+    logger.info(`Events : ${JSON.stringify(scheduledEvents, null, 2)}`)
```

Comment:
I'd say just be a little careful with the logging here. Do we really need it?

Reply:
removed